src/Api/Utilities/CommandResultExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Bit.Api.Models.CommandResults;
+using Bit.Core.Models.Commands;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Utilities;
+
+public static class CommandResultExtensions
+{
+    public static IActionResult MapToActionResult<T>(this CommandResult<T> commandResult)
+    {
+        return commandResult switch
+        {
+            NoRecordFoundFailure<T> => new ObjectResult(commandResult.Errors) { StatusCode = StatusCodes.Status404NotFound },
```

Comment:
I found this [ErrorResponseModel](https://github.com/bitwarden/server/blob/main/src/Core/Models/Api/Response/ErrorResponseModel.cs#L16). Maybe we can map our `CommandResult.Errors` to this model?

Since the model is unique to the HTTP server, we shouldn't add it to the `CommandResult`.